generate-tsconfig.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash 
+# This script generates tsconfig.json file, it looks for all files with extension .ts and adds them to the file section. 
+# Old tsconfig.json file, if exists is saved with .ol.timestamp extension
```

Comment:
Please mention that this script is for testing purposes only and has no impact on typescript toolchain